examples/rosettacode/100_doors.nit

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+
+
+class Door
+	#True : Open / False : Close
+	var state: Bool = false
+
+	fun is_open: Bool
+	do
+		return self.state
+	end
+
+	fun open
+	do
+		self.state = true
+	end
+
+	fun close
+	do
+		self.state = false
+	end
+
+	fun description: String
```

Comment:
You should probably use `redef fun to_s` here, it is used to get a human readable string reprensentation.